--- linux-atm-2.4.1/src/lib/sapequal.c.orig	2001-09-03 20:41:05.000000000 +0200
+++ linux-atm-2.4.1/src/lib/sapequal.c	2004-08-20 13:24:13.916982104 +0200
@@ -64,7 +64,6 @@
 	    CHECK(l2.itu.mode,1);
 	    CHECK(l2.itu.window,a.l2.itu.window > b.l2.itu.window);
 	    break;
-	default:
     }
     switch (a.l3_proto) {
 	case ATM_L3_X25:
@@ -82,7 +81,6 @@
 	case ATM_L3_USER:
 	    if (a.l3.user != b.l3.user) return 0;
 	    break;
-	default:
     }
     return 1;
 }
--- linux-atm-2.4.1/src/sigd/proto.c.orig	2001-09-03 20:41:06.000000000 +0200
+++ linux-atm-2.4.1/src/sigd/proto.c	2004-08-20 13:26:25.705947136 +0200
@@ -258,7 +258,6 @@
 		q_write(&dsc,QF_timer,buf,3);
 		break;
 	    }
-	default:
     }
     va_end(ap);
     if ((size = q_close(&dsc)) >= 0) to_signaling(sock->sig,q_buffer,size);
@@ -287,7 +286,6 @@
 		    q_write(&dsc,QF_ie_id6,&ie,1);
 		}
 		break;
-	    default:
 	}
 	va_end(ap);
     }
--- linux-atm-2.4.1/src/maint/enitune.c.orig	2001-09-03 20:41:06.000000000 +0200
+++ linux-atm-2.4.1/src/maint/enitune.c	2004-08-20 13:27:47.161564000 +0200
@@ -43,7 +43,6 @@
 		mult.rx = strtol(optarg,&end,0);
 		if (*end || mult.rx <= 100) usage(name);
 		break;
-	    default:
 	}
     if (argc != optind+1) usage(name);
     sioc.number = strtol(argv[optind],&end,0);
--- linux-atm-2.4.1/src/qgen/qlib.c.orig	2001-09-03 20:41:05.000000000 +0200
+++ linux-atm-2.4.1/src/qgen/qlib.c	2005-06-06 08:54:09.000000000 +0200
@@ -6,11 +6,7 @@
 #include <config.h>
 #endif
 
-#ifdef DUMP_MODE
-static int q_dump = 0;
-#else
 int q_dump = 0;
-#endif
 
 #ifndef STANDALONE
 #define DUMP qd_dump
@@ -26,7 +22,7 @@
 #include "op.h"
 
 
-static int debug = 0;
+int debug = 0;
 
 
 void PREFIX(report)(int severity,const char *msg,...)
